src/utils/deepLinks.ts

```diff
@@ -2,6 +2,7 @@ import {PROJECT_NAME_PLURAL} from 'src/flows'
 
 export const buildDeepLinkingMap = (orgId: string) => ({
   '/me/about': `/orgs/${orgId}/about`,
```

Comment:
```suggestion
  '/me/about': `/orgs/${orgId}/org-settings`,
```

Reply:
@hoorayimhelping both /about and /org-settings are present here. I thought we agreed to keep both urls for the sake of maintaining backwards compatibility, no?

Reply:
oh just saw your explanation down below.